openedx/core/lib/studio_entities.py

```diff
@@ -0,0 +1,64 @@
+"""Studio entity plugin manager and API."""
+import abc
+
+from openedx.core.lib.api.plugins import PluginManager
+
+
+class StudioEntityPluginManager(PluginManager):
```

Comment:
@andy-armstrong I was following the example set by the course tabs plugin manager, which is also in openedx/core/lib. I was under the impression that new code should be added to openedx/core. Is that not the case?